### ISSUE TITLE
Two-way binding (current value is not displayed)

### ISSUE THREAD

**Issue by novikov1337danil** [sibiraj-s/ngx-tiptap#17]
First of all thanks for your great job.

I tried to implement two way binding with `[(ngModel)]` but the DISPLAYED content of the editor is not updated when its value update was called from the parent component (e.g. set value as an empty string).

To reproduce:
1. Implement two way binding
2. From parent component set any value to ngModel state
![image](https://user-images.githubusercontent.com/44060868/145073806-40bac521-009c-4736-a666-5fdc399e788c.png)
3. `editor.content` has actual state, but not `editor.getHtml()` (as I understand it, it is it who is displayed in the editor)
![image](https://user-images.githubusercontent.com/44060868/145074527-74ee4c91-41fe-4ae2-beb5-26aafb3628ca.png)

I tried to set content manually by `editor.setOptions({content: value })` but editor still display not correct value

**Comment by sibiraj-s** [comment]
> To reproduce
 
A stackblitz will be better than screenshots.

I was able to get the value from the editor with `editor.getHTML()` anytime, which returns the correct value from the editor. 

The editor instance belongs to tiptap core. I am not sure you can set content via `editor.setOptions({content: value })`. Does this work outside angular?

**Comment by novikov1337danil** [comment]
Thanks for the quick response. Here is the link to [reproduce](https://stackblitz.com/edit/ngx-tiptap-kctxvd?file=src/app/app.component.ts)

**Comment by sibiraj-s** [comment]
Something wrong with handling empty values. It's a bug, but not sure its in ngx-tiptap or with tiptap itself. will get this checked. Meanwhile as a workaround you can do this. the binding works fine for other non empty values

```ts
 clear() {
    this.value = '<p></p>';
  }
```

**Comment by sibiraj-s** [comment]
Fixed in 89c510e20f4f1eb42b55b5d0a40e6efc623e97e2

**Comment by sibiraj-s** [comment]
Fixed in v4.0.1. 

v4 requires angular 13, If you are not using 13, you can use `<p></p>` to set empty content which is also a perfectly valid option.

One another option using the editor instance directly is

```ts
editor.chain().setContent('').run()
// or
editor.commands.setContent('')
```

**Comment by novikov1337danil** [comment]
Good job! Thanks for the help
